codebuild/bin/install_awslc_fips.sh

```diff
@@ -50,12 +51,17 @@ INSTALL_DIR=$2
 VERSION=$3
 
 # Map version to a specific feature branch/tag.
+# Note: since the next FIPS validation will be split off from main
+# building main with FIPS enabled is essentially the next FIPS release.
 case $VERSION in
   "2022")
-    AWSLC_BRANCH=AWS-LC-FIPS-2.0.17
+    AWSLC_BRANCH=fips-2022-11-02
     ;;
   "2024")
-    AWSLC_BRANCH=AWS-LC-FIPS-3.0.0
+    AWSLC_BRANCH=fips-2024-09-27
+    ;;
+  "next")
+    AWSLC_BRANCH=main
```

Comment:
I kind of have the opposite concern for this: is it okay to take the latest commit from main, rather than the latest release? I expect the above concerns about breaking our build might actually be relevant for commits outside of a release.

Reply:
I hadn't planned on adding fips-next testing to the PR workflow, only the nightly run, since a next release might be years out, it seems less pressing. 
@andrewhop should we pull the latest release?